deployments/kubernetes/monitoring/prometheus/cluster-role.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Is there a way to only give access to production environments? Cause we really don't care to monitor the other environments. If it's not easy don't worry about it.